.github/workflows/ci.yml

```
@@ -12,28 +11,23 @@ on:
     runs-on: ubuntu-latest
 
-    # needed for miniconda
-    defaults:
-      run:
-        shell: bash -l {0}
-
     steps:
-      - uses: actions/checkout@v3
-      - uses: conda-incubator/setup-miniconda@v2
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)